src/yb/master/backfill_index.cc

```diff
@@ -635,6 +635,27 @@ std::unordered_set<TableId> IndexIdsFromInfos(const std::vector<IndexInfoPB>& in
   return idx_ids;
 }
 
+std::string RetrieveIndexNames(CatalogManager* mgr, const std::unordered_set<std::string>& index_ids) {
```

Comment:
line too long

Reply:
ack

Reply:
btw,  fyi: there is this script https://github.com/yugabyte/yugabyte-db/blob/master/arcanist_util/pre_commit_hook.sh

that you can run locally to reformat your changes in a yb style guide compliant way. 

it's not perfect but gets most of the stylistic guidelines right.